tldw_Server_API/app/api/v1/endpoints/admin/admin_budgets.py

```
@@ -69,3 +70,46 @@ async def admin_update_budget_by_org_id(
         principal=principal,
         db=db,
     )
+
+
+@router.get("/budgets/forecast")
+async def admin_get_budget_forecast(
+    org_id: int = Query(..., description="Organization ID"),
+    principal: AuthPrincipal = Depends(get_auth_principal),
```

Ruff (0.15.7)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)